database/migrations/2020_05_20_100000_create_contacts_table.php

```diff
@@ -22,17 +22,8 @@ public function up()
             $table->string('company_name')->nullable();
             $table->string('phone')->nullable(); // Will need to format before saving
             $table->text('message')->nullable();
-            $table->json('additional_fields')->nullable();
-            
-            // Move application specific data to additonal_fields
-            $table->smallInteger('participants')->nullable();
-            $table->date('requested_date')->nullable();
-            $table->time('requested_time')->nullable(); // Stored in location timezone
-            
-            // Move response fields to different table (contact followup or something like that)
-            $table->dateTime('emailed_at')->nullable();
-            $table->dateTime('closed_at')->nullable();
-            
+            $table->json('fields')->nullable();
```

Comment:
Renamed this to fields based on how it's called in Nova & made a little more sense to me once all of the extra fields were removed. Can change this back if neccesary.

Reply:
Thanks for pointing it out. That works for me!